### DISCUSSION TITLE
Accuracy seems undervalued in lazer pp system

### DISCUSSION THREAD

**Discussion by tutis12** [ppy/osu#26853] [Ideas]
PP values are almost the same as in stable for the same acc.
It should be higher because you need to acc sliders now.
95% acc in the new system would be about 97.5 in stable assuming 50/50 slider/circle object count.

**Comment by sz-da** [comment]
I believe this is planned, but needs further discussion before changing.

**Comment by jamop4** [comment]
IIRC the pp devs already have something in mind (treating sliders as circles when it comes to acc PP) but I don't think anything has been added yet. Would be great once it is added and I think it will be result in more people switching over to Lazer. 

Currently, there isn't much an of an incentive when it comes to farming PP since you are penalized despite it being harder to acc.

**Comment by Rekunan** [comment]
This is one of the bigger lazer pp issues that is planned to be tackled, the main question is how. If you would like to keep up in pp dev, I recommend joining the discord for it: https://discord.gg/aqPCnXu
